tests/by-util/test_cp.rs

```diff
@@ -491,3 +499,220 @@ fn test_cp_no_deref_folder_to_folder() {
     let path_to_check = path_to_new_symlink.to_str().unwrap();
     assert_eq!(at.read(&path_to_check), "Hello, World!\n");
 }
+
```

Comment:
So, all the tests target only "linux"? ☹️

We definitely need to be testing all platforms. If there is some difficulty in getting them to work on MacOS/Windows, I can offer some time to help at a later date.

And, re-reading [the docs](https://doc.rust-lang.org/reference/conditional-compilation.html#target_os), I think `target_os = "unix"` is incorrect; it should be `target_os = "linux"`.

"unix" is a `target_family`.

Reply:
Yeah, I don't know how to fix these :/

Symlinks are tricky on Windows